README.rst

```diff
@@ -918,6 +975,69 @@ configuration status of a device changes, this ensures the check reacts
 quickly to events happening in the network and informs the user promptly
 if there's anything that is not working as intended.
 
+Iperf
+~~~~~
+
+This check provides network performance measurements such as maximum achievable bandwidth, jitter, datagram loss etc of the device using `iperf3 utility <https://iperf.fr/>`_. 
+
+It also supports tuning of various parameters related to timing, buffers and protocols (TCP, UDP with IPv4 and IPv6).
+
+This check is ``disabled`` by **default**, but you may choose to enable auto creation of this check by setting
+`OPENWISP_MONITORING_AUTO_IPERF <#OPENWISP_MONITORING_AUTO_IPERF>`_ to ``True``.
+
+You can also change the default values used for iperf checks using
+`OPENWISP_MONITORING_IPERF_CHECK_CONFIG <#OPENWISP_MONITORING_IPERF_CHECK_CONFIG>`_ setting.
+
+Instructions to configure Iperf Check
+-------------------------------------
+
+1. Register your device to OpenWISP
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Register your device to OpenWISP and make sure `iperf3 openwrt package <https://openwrt.org/packages/pkgdata/iperf3>`_ is installed on the device if not run :
+
+.. code-block:: shell
+
+    opkg install iperf3
+
+2. Enable secure SSH access from OpenWISP to your devices
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We have already well covered this in `How to configure push updates section of openwisp-controller <https://github.com/openwisp/openwisp-controller#how-to-configure-push-updates>`_.
+
+**Note:** Make sure device connection is enabled & working with right update strategy ie. ``OpenWISP SSH``.
+
+.. image:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/enable-openwisp-ssh.png
+  :alt: Enable ssh access from openwisp to device
+  :align: center
+  
+3. Configure Iperf settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Configure iperf servers in `openwisp settings <https://github.com/openwisp/openwisp-monitoring/blob/master/tests/openwisp2/settings.py>`_ , The host can be specified by hostname, IPv4 literal, or IPv6 literal.
+
+For eg.
+
+.. code-block:: python
+
+    OPENWISP_MONITORING_IPERF_SERVERS = {
+    # Public iperf servers also available: https://iperf.fr/iperf-servers.php#public-servers
+    # '<org-pk>': ['<ORG_IPERF_SERVER>']
+    'a9734710-db30-46b0-a2fc-01f01046fe4f': ['speedtest.uztelecom.uz'],
+    'z9734710-db30-46b0-a2fc-01f01046fe4f': ['192.168.5.109'],
+    'c9734710-db30-46b0-a2fc-01f01046fe4f': ['2001:db8::1'],
+    }
+
+4. Run the check
```

Comment:
We should also point it out here to configure the celery beat.

Reply:
Noted